pkgs/development/python-modules/pyqt-distutils/default.nix

```diff
@@ -0,0 +1,22 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+}:
+let
+  pname = "pyqt-distutils";
+  version = "0.7.3";
+in
+buildPythonPackage {
+  inherit pname version;
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-7luMMNOwpHOlwj5DsqKc6OR/p7SNjckYFpU9o3vAU+g=";
+  };
+  meta = {
```

Comment:
Can you add `pythonImportsCheck`? Also add `doCheck = false; # no tests` if not tests are present